src/main/java/com/yugabyte/sample/apps/AppBase.java

```diff
@@ -212,6 +212,15 @@ protected Connection getPostgresConnection(String database, String username, Str
         String connectStr = String.format("%s//%s:%d/%s", url, contactPoint.getHost(),
                 contactPoint.getPort(),
                 database);
+        if (!username.equalsIgnoreCase("yugabyte") || !username.equalsIgnoreCase("postgres")){
```

Comment:
Missed this in the review: The condition should be AND and not OR here. Please correct this with another PR